pvcollada20_schema/pvcollada_schema_0.1.xsd

```diff
@@ -674,6 +681,7 @@
             <xs:documentation xml:lang="en">Timezone offset from UTC, hour. Positive is East of UTC. E.g., for +1:00, enter 1.0</xs:documentation>
           </xs:annotation>
         </xs:element>
+        <!-- I don't understand what is this good for. The exporters should simply represent the 3D geometry so there's no need for this vector -->
         <xs:element name="transformation_vector" type="collada:float3_type">
```

Comment:
I don't understand what is this good for. The exporters should simply represent the 3D geometry so there's no need for this vector

Reply:
The vector is from the geocoordinate lat/long to the origin of the local coordinate system. COLLADA tools don't operate in geocoordinates, so to be able to align the scene to a map or terrain file, this vector is needed.